### ISSUE TITLE
Collection admin unable to approve his own works for a collection

### ISSUE THREAD

**Issue by babaklar** [MESH-Research/commons-team#36]
[A user](https://hcommons.org/members/lloydgraham/) contacted us because he has submitted three works to a collection he created. This is what it looks like when he looks at the Works on his end. <img width="1422" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1UyQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--589b5a179369931e1e9b58862b15503696c6b8cd/KCWorksNoApproval.jpg" alt="KCWorksNoApproval.jpg" />

He is the admin for the collection his is trying to submit them to, and created the group on KC. He does not have a requests option in the nav bar of the collection. Here are more screenshots. 

<img width="1422" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK28yQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f18abb28aac9d9c02ec0fa384bc5262e7ce9a080/Collection%20page%20screenshot.jpg" alt="Collection page screenshot.jpg" />

When I use my super admin permissions, I can see the three works in the collection, waiting for approval. So I'm wondering if this is some sort of bug with his account.

**Comment by monotasker** [comment]
Yeah, it's like he's not actually being recognized with admin privileges. Let me look at his account.

**Comment by monotasker** [comment]
I just sent Lloyd Graham an individual invitation to be an owner of the group. The issue seems to be related to the way his group memberships are set, since roles in linked Commons group collections are assigned to group members as a group, not to individuals. So I'll keep on debugging that underlying issue. But individuals can also be given permissions directly, so for the moment we can get someone quick access that way. He'll just need to accept the invitation I sent.
